js/content.js

```diff
@@ -23,6 +24,8 @@ let checkElement = document.createElement("div");
 var img = document.createElement("img");
 let m_Stream = null;
 
+let intervalCSSId;
```

Comment:
comment telling it what it does (like the rest of the variables above, for consistency)